raiden-dapp/src/components/SplashScreen.vue

```diff
@@ -116,6 +120,24 @@ export default class Loading extends Vue {
   connect(subkey?: true) {
     return subkey;
   }
+
+  async downloadLogs() {
+    const [lastTime, content] = await getLogsFromStore();
+    if (!content) return;
```

Comment:
Probably good idea to notify the user too

Reply:
This is just a debugging option, and doesn't happen if the user has logged at least once in the dApp, which I think is reasonable to assume before the user tries to download the logs, I don't think it's worth it to put a dialog only to tell them that. I can instead allow them to download the file with only a single line of text: `The logs were empty`, so they'll get informed.

Reply:
Or only show the link if there are logs

Reply:
This was my first option, but would require another function just to access and check if there're items in the IndexedDB. If the logs are empty because they never logged in, I'll allow them to download the empty file, what else would they expect?

Reply:
Was just another (and I think cleaner) option but up to you